### DISCUSSION TITLE
Cannot Export Unpublished Changes To A Branch

### DISCUSSION THREAD

**Discussion by kucingtakucin** [community/community#57635] [Codespaces]
### Select Topic Area

Show & Tell

### Body

Hello, I'm having a **problem** when I want to **export unpublished changes to a branch**, why does this button keep loading, I've been waiting for hours but there's no result, the next day I try to click this button again but it keeps loading continuously, even though I only want to export unpublished changes to a branch, how can this be happen? Please help 

![image](https://github.com/community/community/assets/55486364/b5d14dda-ca77-4dc8-9f1f-59630337e854)

**Comment by joshspicer** [comment]
Hi @mbahpoetih, are you having issues connecting to this codespace? I would suggest connecting and manually pushing as a first step, if you can. I've looked it up and there appears to be an issue with this export task on your codespace, but I believe you should be able to connect.

**Reply by joshspicer** [reply]
Also, do you have automatic GPG signing enabled (or have you manually enabled it within your codespace)?

  We're aware of a bug involving exporting branches and GPG signing (in this case you may think of it as a security feature 😉 ).  If that's enabled, disabling (and then manually within the container disabling GPG signing, an actual security feature) may help in this case.

**Comment by chriskthomas** [comment]
You can change/add branches by clicking in the bottom left of the codespace.

Then, commit and push the new branch.
